packages/components/modules/content-feed/web/ContentFeedImage/index.tsx

```diff
@@ -0,0 +1,196 @@
+'use client'
+
+import { ChangeEventHandler, useRef, useState } from 'react'
+
+import { useNotification } from '@baseapp-frontend/utils'
+
+import AddRoundedIcon from '@mui/icons-material/AddRounded'
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded'
+import InsertPhotoOutlinedIcon from '@mui/icons-material/InsertPhotoOutlined'
+import { Box, Typography } from '@mui/material'
+import Image from 'next/image'
+import { Controller, UseFormReturn } from 'react-hook-form'
+
+import {
+  AddFileButton,
+  AddFileWrapper,
+  ContentFeedImageContainer,
+  DropFilesContainer,
+  MiniatureFileWrapper,
+  RemoveFileButton,
+} from './styled'
+
+interface IContentFeedImageProps {
```

Comment:
Lets move this to a types file and then add to the index file.

Also, this types seems a little bit weird. Why is images type equals to never[]?

Reply:
let's also avoid `I` prefix on type interfaces

Reply:
@rodrigonahid @davidbermudez-tsl  can we change that?